apps/postybirb-ui/src/app/app-layout/app-layout.tsx

```diff
@@ -19,8 +19,6 @@ export default function AppLayout() {
       <AppFlyouts />
 
       <EuiPageBody
-        hasBorder={false}
-        hasShadow={false}
```

Comment:
Actually there is no hasBorder or hasShadow props on EuiPageBody so they caused errors